dev/ci/ci-json.sh

```diff
@@ -10,6 +10,6 @@ git_download json
 if [ "$DOWNLOAD_ONLY" ]; then exit 0; fi
 
 ( cd "${CI_BUILD_DIR}/json"
-  make
+  make MENHIRFLAGS=--coq MENHIRFLAGS+=--coq-no-version-check
```

Comment:
```suggestion
  make MENHIRFLAGS="--coq --coq-no-version-check"
```
?

Reply:
Why not, I guess I copied it from Nix where I got into troubles with string quoting but here we should be fine.